recipe-server/client/control/components/recipes/RecipeDetails.js

```diff
@@ -80,6 +80,15 @@ export class ArgumentsValue extends React.PureComponent {
     name: null,
   };
 
+  static stringifyImmutable(value) {
+    return JSON.stringify(value.toJS());
```

Comment:
I think Immutable has some special hooks so that you can just do `JSON.stringify(value)`. It will give the same result as calling `.toJS()` first. If that's true, I don't think we even need this method.

Reply:
>All Immutable Collections also implement toJSON() allowing them to be passed to JSON.stringify directly.

Neat, TIL you can customize the `stringify` output if you implement your own `toJSON`.